skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -142,3 +142,63 @@ def pcoa(distance_matrix):
                              columns=axis_labels),
         proportion_explained=pd.Series(proportion_explained,
                                        index=axis_labels))
+
+
+def pcoa_biplot(ordination, y):
+    """Compute the projection of descriptors into a PCoA matrix
+
+    This implementation is as described in Chapter 9 of Legendre & Legendre,
+    Numerical Ecology 3rd edition.
+
+    Parameters
+    ----------
+    ordination: OrdinationResults
+        The computed principal coordinates analysis of dimensions (n, c) where
+        the matrix ``y`` will be projected onto.
+    y: DataFrame
+        Samples by features table of dimensions (n, m). These can be
+        environmental features or abundance counts. This table should be
+        normalized in cases of dimensionally heterogenous physical variables.
+
+    Returns
+    -------
+    OrdinationResults
+        The modified input object that includes projected features onto the
+        ordination space in the ``features`` attribute.
```

Comment:
An example on how to run this would be great, especially given that this is a fairly new method in this ecosystem -- it isn't exactly clear what proper inputs / outputs should look like.

Reply:
Great idea, I'll add something.